integration-tests/src/env/test_env.rs

```diff
@@ -90,6 +90,11 @@ impl TestEnv {
         TestEnvBuilder::new(genesis_config.clone())
     }
 
+    pub fn default_builder_with_genesis() -> TestEnvBuilder {
+        let genesis = Genesis::test(vec!["test0".parse().unwrap()], 1);
+        TestEnvBuilder::from_genesis(genesis.clone())
```

Comment:
you don't need `.clone()` here

Reply:
Removed in [a036d1a](https://github.com/near/nearcore/pull/13091/commits/a036d1a85297f2dd59356fdc7fa550395edce1ee)